app/streaming/input.cpp

```diff
@@ -775,9 +775,20 @@ void SdlInputHandler::handleControllerDeviceEvent(SDL_ControllerDeviceEvent* eve
         state->jsId = SDL_JoystickInstanceID(SDL_GameControllerGetJoystick(state->controller));
         state->haptic = SDL_HapticOpenFromJoystick(SDL_GameControllerGetJoystick(state->controller));
```

Comment:
It looks like you changed the file permissions on input.cpp from 644 to 755. Please put them back to 644.

Reply:
Fixed